integration-tests/utils/integration-utils.js

```diff
@@ -190,9 +190,27 @@ const dismissNotifications = async page => {
 }
 
 const signIntoTerra = async (page, { token, testUrl }) => {
-  !!testUrl && await page.goto(testUrl, waitUntilLoadedOrTimeout(60 * 1000))
-  await page.waitForXPath('//*[contains(normalize-space(.),"Loading Terra")]', { hidden: true, timeout: 60 * 1000 })
-  await waitForNoSpinners(page)
```

Comment:
An issue with waiting for spinner during page loading is that we can "miss" the spinner when it has finished checking before it renders.

Reply:
@aweng98 IIUC the issue you describe in your comment above, could wrapping the original code in a [noSpinnersAfter](https://github.com/DataBiosphere/terra-ui/blob/e69798efefbadf368f737876d84fd8358b45f6ad/integration-tests/utils/integration-utils.js#L166) suffice?

Reply:
No, I don't think that helps. The best way (IMO) is by finding a visible UI element (or more) first to verify page has rendered then checks and waits for running spinner (ajax loading).

Reply:
I don't think it will work because page.goto changes the page, and `noSpinnersAfter` checks that a spinner appears and disappear within the same page.